spec/helpers/web_helpers.rb

```diff
@@ -0,0 +1,21 @@
+
+
+def sign_up
+  FactoryGirl.create(:hipster)
+end
+
+def sign_in(user)
+  click_link "Sign in"
+  fill_in "hipstername", with: user.hipstername
+  fill_in "password", with: user.password
+  click_button "Log in"
+end
+
+
+def add_picture
+    visit '/hipstergrams'
+    click_link 'Upload new hipstergram'
+    attach_file :hipstergram_image, './spec/photos/test.jpg'
+    fill_in 'Description', with: 'Just some triple distilled mason water cold brew'
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)